docs/BreakingChanges/022 Some WorkFlow drag and drop APIs are obsolete.md

```diff
@@ -0,0 +1,25 @@
+## 22: Some WorkFlow drag/drop APIs are obsolete
```

Comment:
{
  "Id": "22",
  "Title": "Some WorkFlow drag/drop APIs are obsolete",
  "Details": "This WorkFlow Drag/Drop API is obsolete and will cause compiler warnings if the app is rebuilt against 4.5.",
  "Suggestion": "New [DragDropHelper](https://msdn.microsoft.com/en-us/library/system.activities.presentation.dragdrophelper(v=vs.110).aspx) APIs that support operations with multiple objects should be used instead. Alternatively, the build warnings can be suppressed or they can be avoided by using an older compiler. The APIs are still supported.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887#wwf",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Activities.Presentation.DragDropHelper.DoDragMove(System.Activities.Presentation.WorkflowViewElement,System.Windows.Point)",
    "M:System.Activities.Presentation.DragDropHelper.GetCompositeView(System.Windows.DragEventArgs)",
    "M:System.Activities.Presentation.DragDropHelper.GetDraggedModelItem(System.Windows.DragEventArgs)",
    "M:System.Activities.Presentation.DragDropHelper.GetDroppedObject(System.Windows.DependencyObject,System.Windows.DragEventArgs,System.Activities.Presentation.EditingContext)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Activities.Presentation.DragDropHelper.DoDragMove(System.Activities.Presentation.WorkflowViewElement,System.Windows.Point)
- [x] M:System.Activities.Presentation.DragDropHelper.GetCompositeView(System.Windows.DragEventArgs)
- [x] M:System.Activities.Presentation.DragDropHelper.GetDraggedModelItem(System.Windows.DragEventArgs)
- [x] M:System.Activities.Presentation.DragDropHelper.GetDroppedObject(System.Windows.DependencyObject,System.Windows.DragEventArgs,System.Activities.Presentation.EditingContext)